src/helm/benchmark/run_specs/vlm_run_specs.py

```diff
@@ -836,6 +836,75 @@ def get_pairs_spec(subset: str, person: str) -> RunSpec:
     )
 
 
+@run_spec_function("fair_face")
+def get_fair_face_spec(attribute: str, subgroup: str) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.vision_language.fair_face_scenario.FairFaceScenario",
+        args={"attribute": attribute, "subgroup": subgroup},
+    )
+    adapter_spec: AdapterSpec = _get_multiple_choice_joint_adapter_spec(
+        input_noun=None,
+        output_noun="Answer",
+        num_outputs=1,
+        max_train_instances=0,
+    )
+    metric_specs: List[MetricSpec] = get_exact_match_metric_specs()
+
+    run_spec_name: str = "fair_face"
+    return RunSpec(
+        name=f"{run_spec_name}:attribute={attribute},subgroup={subgroup}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=metric_specs,
+        groups=[run_spec_name],
```

Comment:
Add `"fair_face"` to `groups`?

Reply:
I have this line so `run_spec_name: str = "fair_face"`, so `fair_face` is already in groups.